.github/actions/install-wdk/action.yml

```diff
@@ -130,7 +134,16 @@ runs:
           Write-Output "WindowsSdkBinPath=$packages\Microsoft.Windows.SDK.CPP.$version\c\bin" >> $env:GITHUB_ENV
         } else {
 
-          Write-Host "Using Winget to install WDK version $inputVersion..."
+          Write-Host "Using Winget to install WDK and SDK version $inputVersion..."
+
+          # Install Windows SDK 10.0.22621 explicitly when WDK 10.0.22621 is requested
```

Comment:
Comment should explain _why_ we are installing this version explicilty (i.e. because it's not present on the runner)

Reply:
This comment is removed in the latest code. Closing this comment.